OpenBCI_GUI/OpenBCI_GUI.pde

```diff
@@ -366,11 +370,56 @@ void setup() {
   buttonHelpText = new ButtonHelpText();
 
   myPresentation = new Presentation();
+  
+  // Setup the UDP receiver 
+  int portRX = 51000;  // the UDP receiving port
+  String ip = "127.0.0.1";  // localhost for now
+  
+  //create new object for receiving 
+  udpRX=new UDP(this,portRX,ip);
```

Comment:
Please have space between variable name and equal sign and space between equal sign and new. You also must  put spaces after commas in function arguments.